charts/retool/templates/deployment_backend.yaml

```diff
@@ -270,6 +270,12 @@ spec:
           successThreshold: {{ .Values.readinessProbe.successThreshold }}
           periodSeconds: {{ .Values.readinessProbe.periodSeconds }}
 {{- end }}
+{{- if .Values.preStopHook.enabled }}
+        lifecycle:
+          preStop:
+            exec:
+              command: ["sleep", "30"]
```

Comment:
I thought this only needed to be 15 secs?

Reply:
wanted to wait for longer in-flight requests